metrics/prometheus/prometheus.yml

```diff
@@ -0,0 +1,6 @@
+scrape_configs:
+  - job_name: myapp
+    scrape_interval: 10s
+    metrics_path: /metrics
+    static_configs:
+      - targets: ["host.docker.internal:2112"]
```

Comment:
What is this host? Is this using the docker network?

Reply:
It's syntax for how docker for windows exposes localhost. I'll add conventions for mac and linux (which just uses the normal localhost format)